main/templates/footer.html

```diff
@@ -51,11 +51,10 @@ <h2><a href="https://mitbootcamps.zendesk.com/hc/en-us/requests/new">Contact Us<
         </div>
         <div class="col-md-4">
           <div class="address-area">
-            <h3>MIT Bootcamps</h3>
+            <img class="open-learning-logo-img" src="/static/images/mit-ol-logo.svg" alt="MIT Open Learning Logo" />
```

Comment:
The spacing between the logo and address should be increased:
<img width="283" alt="image" src="https://github.com/mitodl/bootcamp-ecommerce/assets/34372316/3dff53aa-edc3-4821-8e31-d306d3c96b7e">

Reply:
Updated in https://github.com/mitodl/bootcamp-ecommerce/pull/1484/commits/0504a2bf7756fff2b18460a0245694069f4c3cc2
![Screenshot 2024-02-13 at 3 49 03 PM](https://github.com/mitodl/bootcamp-ecommerce/assets/52656433/2ea70ca0-b6a3-47b9-b834-e21afe78e647)